games-arcade/performous/performous-1.1-r2.ebuild

```diff
@@ -1,18 +1,15 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
 
-CMAKE_REMOVE_MODULES="yes"
-CMAKE_REMOVE_MODULES_LIST="FindALSA FindBoost FindFreetype FindGettext FindJpeg FindPng FindTiff FindZ"
-inherit cmake-utils desktop gnome2-utils
-
-MY_PN="Performous"
-MY_P="${MY_PN}-${PV}"
-SONGS_PN="ultrastar-songs"
+CMAKE_REMOVE_MODULES_LIST=( FindALSA FindBoost FindFreetype FindGettext FindJpeg FindPng FindTiff FindZ )
```

Comment:
not that it's super important: but does this have to be here? This doesn't affect metadata generation or other global scope thingies, so it could just be with `PATCHES` and friends?

Reply:
I prefer listing of cmake meta variables pre-inherit